src/test/scala/ai/eto/rikai/sql/spark/execution/CreateModelCommandTest.scala

```diff
@@ -38,16 +38,18 @@ class CreateModelCommandTest extends AnyFunSuite with SparkTestSession {
   test("create model with options") {
     spark
       .sql(
-        "CREATE MODEL model_options OPTIONS (foo='bar',num=1.2,flag=True) USING 'test://foo'"
+        "CREATE MODEL qualified_opt OPTIONS (" +
+          "rikai.foo.bar=1.23, foo='bar', num=1.23, flag=True) " +
```

Comment:
does this support quoting? what if a function argument collides with a SQL keyword?

Reply:
it supports backquoted identifier.

https://github.com/eto-ai/rikai/blob/master/src/main/antlr4/org/apache/spark/sql/ml/parser/RikaiExtSqlBase.g4#L34-L38